frontend/src/api/constants/index.js

```diff
@@ -1,37 +1,31 @@
-const BASE_URL = process.env.REACT_APP_API_BASE;
-export const BASE_MEDIA_URL = process.env.REACT_APP_MEDIA_API_BASE;
+export const BASE_URL = process.env.REACT_APP_API_BASE;
 
 const apiConstants = {
-  base: BASE_URL,
-  media: BASE_MEDIA_URL,
-  blogs: "blogs",
-  events: "events",
-  team: "team",
-  mentors: "mentors/",
-  mentorAchievements: "mentors/achievements/",
-  mentorIntern: "mentors/intern/",
-  mentorsDocs: "/documents/mentor-docs",
-  faqs: "faq",
-  branch: "branch",
+  pages: BASE_URL + "/pages/",
+  mentors: BASE_URL + "/mentors/",
+  docs: BASE_URL + "/documents/",
+  blogs: "blogs/",
+  events: "events/",
+  team: "team/",
+  mentorsDocs: "mentor-docs/",
+  faqs: "faq/",
+  branch: "branch/",
   interests: "interests/",
-  groups: "campusGroups",
-  blogcategory: "blogCategory",
+  groups: "campusGroups/",
```

Comment:
Use kebab-case for URL routes.
```suggestion
  groups: "campus-groups/",
```